tests/core/test_float_builtins.c

```diff
@@ -1,17 +1,22 @@
 #include <stdio.h>
-#include <math.h>
+
+#define TEST(op)                    \
```

Comment:
This needs a comment saying it tests the `f`, `l` and double version of each operator.